meta-balena-warrior/recipes-core/volatile-binds/volatile-binds/mount-copybind

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+#
+# Perform a bind mount, copying existing files as we do so to ensure the
+# overlaid path has the necessary content.
+
+if [ $# -lt 2 ]; then
+    echo >&2 "Usage: $0 spec mountpoint [OPTIONS]"
+    exit 1
+fi
+
+# e.g. /var/volatile/lib
+spec=$1
+
+# e.g. /var/lib
+mountpoint=$2
+
+if [ $# -gt 2 ]; then
+    options=$3
+else
+    options=
+fi
+
+[ -n "$options" ] && options=",$options"
+
+mkdir -p "${spec%/*}"
+
+if [ -d "$mountpoint" ]; then
+
+    if [ -d "$spec" ]; then
+        specdir_existed=yes
+    else
+        specdir_existed=no
+        mkdir "$spec"
+    fi
+
+    # Fast version of calculating `dirname ${spec}`/.`basename ${spec}`-work
+    overlay_workdir="${spec%/*}/.${spec##*/}-work"
+    mkdir "${overlay_workdir}"
+
+    # Try to mount using overlay, which is must faster than copying files.
+    # If that fails, fall back to slower copy.
+    # FIXME:
+    # Avoid overlayfs mounts as the rootfs is a aufs on some devices which
+    # might break when reading files from lower directory.
+    #if ! mount -t overlay overlay -olowerdir="$mountpoint",upperdir="$spec",workdir="$overlay_workdir" "$mountpoint" > /dev/null 2>&1; then
+
+        if [ "$specdir_existed" != "yes" ]; then
+            cp -pPR "$mountpoint"/. "$spec/"
```

Comment:
The original has a `cp -aPR` instead of a `cp -pPR`

Any reason for this particular diff?

Reply:
I only commented the overlayfs mount. It could be that a later commit in poky changed and you compared with another rev.

Reply:
yea. seems like a recent commit http://cgit.openembedded.org/openembedded-core/commit/meta/recipes-core/volatile-binds/files/mount-copybind?id=ab7e3caf3e29f2d8db55e2901e0a9b01061afd88